pkg/webservice/handlers_test.go

```diff
@@ -2602,3 +2607,82 @@ func NewResponseRecorderWithDeadline() *ResponseRecorderWithDeadline {
 		ResponseRecorder: httptest.NewRecorder(),
 	}
 }
+
+func TestCompressGetQueueApplicationAPI(t *testing.T) {
+	schedulerContext, err := scheduler.NewClusterContext(rmID, policyGroup, []byte(configDefault))
+	assert.NilError(t, err, "Error when creating cluster context.")
+	partitionName := common.GetNormalizedPartitionName("default", rmID)
+	part := schedulerContext.GetPartition(partitionName)
+	_ = addApp(t, "app-1", part, "root.default", false)
+	_ = addApp(t, "app-2", part, "root.default", false)
+	_ = addApp(t, "app-3", part, "root.default", false)
+
+	m := NewWebApp(schedulerContext, nil)
+	m.StartWebApp()
+	defer func() {
+		err = m.StopWebApp()
+		assert.NilError(t, err, "Error when closing webapp service.")
+	}()
+
+	err = common.WaitFor(500*time.Millisecond, 2*time.Second, func() bool {
+		conn, connErr := net.DialTimeout("tcp", net.JoinHostPort("127.0.0.1", "9080"), time.Second)
```

Comment:
It's safer to immediately call `conn.Close()` if possible because the connection is not needed (this is also more readable):
```
conn, connErr := net.DialTimeout("tcp", net.JoinHostPort("127.0.0.1", "9080"), time.Second)
if conn != nil {
  conn.Close()
}
return connErr == nil
```

Reply:
Thanks for the advice. Done.